### PR TITLE
IGAPP-663: Details tag not enough Height

### PR DESCRIPTION
**Pull request by Taggotty**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: native/src/utils/renderHtml.ts [hunk 1]
```diff
@@ -77,6 +77,10 @@ const renderJS = (cacheDictionary: Record<string, string>) => `
   
   window.addEventListener('load', adjustHeight);
   window.addEventListener('resize', adjustHeight);
+  var details = document.querySelectorAll("details")
```
Comment (f1sh1918) [inline native/src/utils/renderHtml.ts:80]: a comment would be nice :)

File: native/src/utils/renderHtml.ts [hunk 2]
```diff
@@ -77,6 +77,10 @@ const renderJS = (cacheDictionary: Record<string, string>) => `
...
+  details.forEach((detail => {
```
Comment (f1sh1918) [inline native/src/utils/renderHtml.ts:81]:
```suggestion
  details.forEach((detail => detail.addEventListener("toggle", adjustHeight)))
```

Comment (f1sh1918) [reply native/src/utils/renderHtml.ts:81]: no need for curly brackets

File: release-notes/unreleased/IGAPP-663-Details-Tag-dynamic-height.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+issue_key: IGAPP-663
+show_in_stores: true
+platforms: # relevant platforms, possible values: web, android and ios
+  - android
+  - ios
+en: The content height is now adjusted dynamically when expanding and collapsing details.
+de: Der höhe von Inhalten wird nun dynamisch angepasst, wenn Details geöffnet oder geschlossen werden.
```
Comment (f1sh1918) [inline release-notes/unreleased/IGAPP-663-Details-Tag-dynamic-height.yml:7]:
```suggestion
de: Der Höhe von Inhalten wird nun dynamisch angepasst, wenn Details geöffnet oder geschlossen werden.
```

File: release-notes/unreleased/IGAPP-663-Details-Tag-dynamic-height.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+issue_key: IGAPP-663
+show_in_stores: true
+platforms: # relevant platforms, possible values: web, android and ios
+  - android
+  - ios
+en: The content height is now adjusted dynamically when expanding and collapsing details.
```
Comment (f1sh1918) [inline release-notes/unreleased/IGAPP-663-Details-Tag-dynamic-height.yml:6]:
```suggestion
en: The content height is now adjusted dynamically when expanding or collapsing details.
```

**Review by f1sh1918 — APPROVED** [review]

File: release-notes/unreleased/IGAPP-663-Details-Tag-dynamic-height.yml (hunk 3 above)
Comment (Znippet) [reply release-notes/unreleased/IGAPP-663-Details-Tag-dynamic-height.yml:7]: Even better: Die Höhe ;)

**Review by Znippet — APPROVED** [review]

**Review by klinzo — APPROVED** [review]
Tested on android :+1:

**Pull request merged**